bdb/lite.c

```diff
@@ -668,16 +700,13 @@ int bdb_lite_list_records(bdb_state_type *bdb_state,
                                           int *bdberr),
                           int *bdberr)
 {
-    DB *db;
     DBC *dbcp;
     DBT key, data;
     int rc = 0;
 
     *bdberr = 0;
 
-    db = bdb_state->dbp_data[0][0];
-
-    rc = db->cursor(db, NULL, &dbcp, 0);
+    rc = get_cursor_lite(bdb_state, NULL, &dbcp);
```

Comment:
Same here -- we should close always after c_get()?

Reply:
Wait, I think those get closed at the end of the function, no?